pybamm/input/parameters/lithium-ion/cathodes/LFP_Prada2013/LFP_electrolyte_exchange_current_density_kashkooli2017.py

```diff
@@ -0,0 +1,37 @@
+from pybamm import exp, constants, Parameter

```

Comment:
I don't understand the folder structure for this parameter set. How come Prada 2013 has parameter sets from 2017?